project.mml

```diff
@@ -1646,6 +1646,8 @@ Layer:
                                'laundry', 'dry_cleaning', 'beverages', 'perfumery', 'cosmetics', 'variety_store', 'wine', 'outdoor', 
                                'copyshop', 'sports', 'deli', 'tobacco', 'art', 'tea', 'coffee') THEN shop
                                ELSE 'other' END AS shop,
+            name,
```

Comment:
The diff of the changes in this file is a bit hard to read, what do we change exactly in this file?

Is it still necessary to have a separate text layer in addition to the amenity-points layer?